### DISCUSSION TITLE
Indicatore: Aggregati strutturali a rischio frane

### DISCUSSION THREAD

**Discussion by carlaiadanza** [pcm-dpc/DPC-Aggregati-Strutturali-ITI-Centro#8] [Show and tell]
L'edizione 2021 del Rapporto ISPRA su _Dissesto idrogeologico in Italia: pericolosità e indicatori di rischio_ presenta un nuovo indicatore relativo agli Aggregati strutturali a rischio frane:
https://www.isprambiente.gov.it/it/pubblicazioni/rapporti/dissesto-idrogeologico-in-italia-pericolosita-e-indicatori-di-rischio-edizione-2021

**Comment by pierluigicara** [comment]
Mi pare una bella notizia sotto diversi punti di vista. Oltre ad essere un bell'esempio di riutilizzo offerto dalla disponibilità di open data (di qualità), potrà essere un ulteriore supporto per le politiche pubbliche nel settore del dissesto idrogeologico.

**Comment by Manuele85** [comment]
Buon pomeriggio,
per poter visualizzare i dati dei vari comuni, quindi aprire i file:
- .dbf
- .prj
- .shp
Quale programma posso utilizzare?
per i file .dbf ho notato che con ACCESS li apre e si vedono in formato elenco i dati IDAGR + Label + comune
Gli altri due file .prj e .shp, quali dati contengono e come aprirli?

Complimenti per il lavoro
Grazie
Un caro saluto
Manuele

**Comment by pierluigicara** [comment]
Buongiorno @manuele85
per aprire e utilizzare i file serve un sw GIS (QGIS e' quello open più diffuso).
Io ormai sono in pensione da un po' e in caso di ulteriori necessità consiglio di contattare direttamente il DPC (anche tramite Contact Center).
Saluti
Pierluigi



Inviato dal mio Galaxy



-------- Messaggio originale --------
Da: Manuele85 ***@***.***>
Data: 04/03/25 16:02 (GMT+01:00)
A: pcm-dpc/DPC-Aggregati-Strutturali-ITI-Centro ***@***.***>
Cc: "Cara Pierluigi (Consulenti)" ***@***.***>, Comment ***@***.***>
Oggetto: Re: [pcm-dpc/DPC-Aggregati-Strutturali-ITI-Centro] Indicatore: Aggregati strutturali a rischio frane (Discussion #8)


Buon pomeriggio,
per poter visualizzare i dati dei vari comuni, quindi aprire i file:

  *   .dbf
  *   .prj
  *   .shp
Quale programma posso utilizzare?
per i file .dbf ho notato che con ACCESS li apre e si vedono in formato elenco i dati IDAGR + Label + comune
Gli altri due file .prj e .shp, quali dati contengono e come aprirli?

Complimenti per il lavoro
Grazie
Un caro saluto
Manuele

**Reply by Manuele85** [reply]
In primis grazie per la tua celere risposta, encomiabile anche il lavoro fatto!

Quindi, con QGIS, in pratica è come aprire le tre estensioni per ogni singolo comune, ma da un solo software?
Io per ora avevo provato:
- aprendo il file DB con ACCESS, e visualizzavo ovviamente in forma elenco
- aprendo il file . shp con Google Earth Pro, quindi interattivo per il comune selezionato.shp (ma i dati erano i medesimi del file DB)
- mi manca il file . prj (non mi è chiaro se ci sono anche eventualmente altri dati visualizzabili)...

Grazie
